frontend/src/metabase/writeback/utils.ts

```diff
@@ -67,7 +67,7 @@ export const isActionButtonCard = (card: SavedCard) =>
 export function isActionButtonDashCard(
   dashCard: BaseDashboardOrderedCard,
 ): dashCard is ActionButtonDashboardCard {
-  const virtualCard = dashCard.visualization_settings?.virtual_card;
+  const virtualCard = dashCard?.visualization_settings?.virtual_card;
   return isActionButtonCard(virtualCard as SavedCard);
 }
```

Comment:
Curios what should happen so a null-ish `dashCard` gets passed here? Might make sense to also type it like `dashCard?: BaseDashboardOrderedCard` to enforce type-safety

Reply:
not sure how this happened, but I was running into crashes here, maybe on an initial render it was sending through a null dashcard?